README.md

```diff
@@ -4,15 +4,19 @@ The Dockerfiles in this repository are used for building the .NET Core product.
 
 ## Where are the published images
 
-The images produced from the Dockerfiles are published to the [microsoft/dotnet-buildtools-prereqs](https://hub.docker.com/r/microsoft/dotnet-buildtools-prereqs/) Docker Hub repository.
+Each image produced from the Dockerfiles is published based on its repository in the [manifest](#manifest):
 
-- [Most recent tags](https://hub.docker.com/r/microsoft/dotnet-buildtools-prereqs/tags/)
-- [Full list of tags](https://registry.hub.docker.com/v1/repositories/microsoft/dotnet-buildtools-prereqs/tags)
+- [microsoft/dotnet-buildtools-prereqs](https://hub.docker.com/r/microsoft/dotnet-buildtools-prereqs/) is a Docker Hub repository.
+  - [Most recent tags](https://hub.docker.com/r/microsoft/dotnet-buildtools-prereqs/tags/)
+  - [Full list of tags](https://registry.hub.docker.com/v1/repositories/microsoft/dotnet-buildtools-prereqs/tags)
+- **dotnet-buildtools-prereqs** is a repository in the *DotnetDocker* private Azure Container Registry.
```

Comment:
@MichaelSimons Do you know if there's a good way to link to the resource? (Without confusing people with 404s, etc.?)

Reply:
I'm not aware of one.